server/datastore/mysql/migrations/tables/20161118212613_CreateTableScheduledQueries.go

```diff
@@ -10,7 +10,7 @@ func init() {
 
 func Up_20161118212613(tx *sql.Tx) error {
 	_, err := tx.Exec(
-		"CREATE TABLE `scheduled_queries` (" +
+		"CREATE TABLE IF NOT EXISTS `scheduled_queries` (" +
```

Comment:
This is something I changed before I realized that the weirdness might be caused because we were writing the schema halfway through the tests running. I can revert, but felt like it wasn't bad.

Reply:
I think `IF NOT EXISTS` is actually a good practice wherever possible in MySQL migrations, as MySQL automatically commits after DDL statements, so the `CREATE TABLE` will try to run again if the migration fails on some step after this. Unfortunately the same can't be done for columns.